Examples/Fermions/fermi_hubbard.py

```diff
@@ -0,0 +1,91 @@
+import netket as nk
+import numpy as np
+import matplotlib.pyplot as plt
+import json
+import jax
+
+from netket import experimental as nkx
+
+L = 2  # take a 2x2 lattice
+D = 2
+t = 1  # tunneling/hopping
+U = 0.01  # coulomb
+
+# create the graph our fermions can hop on
+g = nk.graph.Hypercube(length=L, n_dim=D, pbc=True)
+Nsites = g.n_nodes
```

Comment:
```suggestion
n_sites = g.n_nodes
```
the formatting should fail here I think?